test-app/.circleci/config.yml

```diff
@@ -1,7 +1,15 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.2
+  hmpps: ministryofjustice/hmpps@3.8
```

Comment:
A few changes here and below to keep the test-app up to date with hmpps-template-kotlin